includes/Interfaces/Product_Query.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Interface Product_Query
+ *
+ * @link      https://github.com/googleforcreators/web-stories-wp
+ *
+ * @copyright 2022 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ */
+
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Interfaces;
+
+use Google\Web_Stories\Model\Product;
+use WP_Error;
+
+/**
+ * Interface Product_Query.
+ */
+interface Product_Query {
+	/**
+	 * Get products by search term.
+	 *
+	 * @since 1.21.0
+	 *
+	 * @param string $search_term Search term.
+	 * @return Product[]|WP_Error
+	 */
+	public function get_search( string $search_term );
```

Comment:
We need sorting in the very near future. Is the thinking to simply add two additional `$orderby` and `$order` parameters? That works for me, just want to clarify that.